tuta-sdk/rust/sdk/src/instance_mapper.rs

```diff
@@ -564,36 +563,38 @@ impl Serializer for ElementValueSerializer {
 		Ok(ElementValue::Bool(v))
 	}
 
-	fn serialize_i8(self, _: i8) -> Result<Self::Ok, Self::Error> {
-		unsupported("i8")
+	fn serialize_i8(self, v: i8) -> Result<Self::Ok, Self::Error> {
```

Comment:
we don't have most number sized. I'm not sure if we should write code for them or leave them explicitly unsupported.

Reply:
sorry, I meant we don't support most numbers

Reply:
True. maybe because we were using some random test Entity with `u8/i8` field. should not be part of this commit anyway